working_copy/machinev1

```diff
@@ -238,7 +243,7 @@
           "value": "undetermined"
         }
       ],
-      "predicate": "other"
+      "predicate": "accidental event"
```

Comment:
Why is the "other" taxonomy renamed? that's a major change with big impact

Reply:
It's not renamed. I copied the part to introduce "Accidental Event" Category including "Misconfiguration" example (takem from "Availability")

Reply:
You created *accidental event* by copying it, yes. But you also modified the other-taxonomy as these two lines clearly show.